docs/base/part_properties.rst

```diff
@@ -311,6 +311,15 @@ The packages to install in the staging area for deployment with the build
 products. These provide libraries and executables to support the deployed
 part.
 
+For Rockcraft:
```

Comment:
It works regardless of the application in use, any special need to mention Rockcraft? If this is valid only for certain applications we can say so and refer to specific application documentation.

Reply:
OK, I removed Rockcraft, it was said only for rockcraft in its docs.